src/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-﻿project (Cuberite)
+project (Cuberite)
```

Comment:
What's the change here? I don't see anything.

Reply:
I think this is removing a [UTF8 BOM](https://github.com/cuberite/cuberite/pull/3936#issuecomment-325988099) that got added at some point.